app/graphql/types/query_type.rb

```diff
@@ -44,26 +44,35 @@ def record_id(id:, index:)
 
       field :search, SearchType, null: false,
                                  description: 'Search for timdex records' do
-        argument :searchterm, String, required: false, default_value: nil
-        argument :citation, String, required: false, default_value: nil
-        argument :contributors, String, required: false, default_value: nil
-        argument :funding_information, String, required: false, default_value: nil
-        argument :identifiers, String, required: false, default_value: nil
-        argument :locations, String, required: false, default_value: nil
-        argument :subjects, String, required: false, default_value: nil
-        argument :title, String, required: false, default_value: nil
-        argument :from, String, required: false, default_value: '0'
+        argument :searchterm, String, required: false, default_value: nil, description: 'Query all searchable fields'
+        argument :citation, String, required: false, default_value: nil, description: 'Search by citation information'
+        argument :contributors, String, required: false, default_value: nil, description: 'Search by contributor names'
```

Comment:
What do you think about something like `Search by contributor/author names` to make it clearer what contributor means?

Reply:
For sure. I might tweak it a bit further to provide a couple contributor examples, like "e.g., author names, editor names, etc."